### DISCUSSION TITLE
Why does IXP session lack the BGP-GROUP and RELATIONSHIP functionality ?

### DISCUSSION THREAD

**Discussion by emil-palm** [peering-manager/peering-manager#895] [Q&A]
Like the topic goes, direct sessions has two fields; BGP-GROUP and RELATIONSHIP. Why are these missing from IXP sessions?

/ emil

**Comment by gmazoyer** [comment]
An IXP in Peering Manager is represented as a specific BGP group. Since IXP sessions are linked to IXPs, they are always associated with groups. IXPs are specialised groups meant to represent public peerings. Therefore, relationships were not added to sessions belonging to IXPs, as it was assumed that these relationships would always be the same.
